<?xml version="1.0" encoding="UTF-8"?>
<ui version="6.0">
 <class>mashEditor</class>
 <widget class="QDialog" name="mashEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mash Editor</string>
  </property>

  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget_editor">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_main">
      <attribute name="title">
       <string>Main</string>
      </attribute>

      <layout class="QGridLayout" name="gridLayoutMain">
       <item row="0" column="1">
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="4">
        <widget class="QLineEdit" name="lineEdit_name">
         <property name="toolTip">
          <string>Name</string>
         </property>
        </widget>
       </item>

       <item row="2" column="1">
        <widget class="SmartLabel" name="label_grainTemp">
         <property name="text">
          <string>Initial grain temp</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_grainTemp</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="SmartLineEdit" name="lineEdit_grainTemp">
         <property name="toolTip">
          <string>Initial grain temp</string>
         </property>
        </widget>
       </item>

       <item row="4" column="1">
        <widget class="SmartLabel" name="label_spargeTemp">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Sparge temp</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_spargeTemp</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="SmartLineEdit" name="lineEdit_spargeTemp">
         <property name="toolTip">
          <string>Sparge temp target</string>
         </property>
        </widget>
       </item>

       <item row="6" column="1">
        <widget class="SmartLabel" name="label_spargePh">
         <property name="text">
          <string>Sparge pH</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_spargePh</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="SmartLineEdit" name="lineEdit_spargePh">
         <property name="toolTip">
          <string>Sparge pH</string>
         </property>
        </widget>
       </item>

       <item row="13" column="0">
        <widget class="InfoButton" name="infoButton_id"></widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="label_id">
         <property name="text">
          <string>ID in database</string>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QLabel" name="label_id_value">
        </widget>
       </item>
       <item row="14" column="1" colspan="5">
        <widget class="InfoText" name="infoText_id">
         <property name="buddy">
          <cstring>infoButton_id</cstring>
         </property>
         <property name="text">
          <string>You can normally ignore this, but it's sometimes useful for debugging.</string>
         </property>
        </widget>
       </item>

      </layout>
     </widget>

     <widget class="QWidget" name="tab_tun">
      <attribute name="title">
       <string>Use for</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutTun">
       <item row="0" column="1">
        <widget class="SmartLabel" name="label_tunTemp">
         <property name="text">
          <string>Initial tun temp</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_tunTemp</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="SmartLineEdit" name="lineEdit_tunTemp">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>

       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_fromEquipment">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Get following parameters from the recipe's equipment.</string>
         </property>
         <property name="text">
          <string>From Equipment</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>

       <item row="4" column="1">
        <widget class="SmartLabel" name="label_tunMass">
         <property name="text">
          <string>Tun mass</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_tunMass</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="SmartLineEdit" name="lineEdit_tunMass">
         <property name="toolTip">
          <string>Tun mass</string>
         </property>
        </widget>
       </item>

       <item row="6" column="1">
        <widget class="SmartLabel" name="label_tunSpHeat">
         <property name="text">
          <string>Tun specific heat</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_tunSpHeat</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="SmartLineEdit" name="lineEdit_tunSpHeat">
         <property name="toolTip">
          <string>Tun specific heat (cal/(g*K))</string>
         </property>
        </widget>
       </item>

      </layout>
     </widget>

     <widget class="QWidget" name="tab_notes">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="textEdit_notes"/>
       </item>
      </layout>
     </widget>

    </widget>
   </item>

   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_Buttons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_new">
       <property name="toolTip">
        <string>New</string>
       </property>
       <property name="text">
        <string>New</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/images/smallPlus.svg</normaloff>:/images/smallPlus.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="toolTip">
        <string>Save and close</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/images/filesave.svg</normaloff>:/images/filesave.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="toolTip">
        <string>Discard and close</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/images/exit.svg</normaloff>:/images/exit.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InfoButton</class>
   <extends>QPushButton</extends>
   <header>widgets/InfoButton.h</header>
  </customwidget>
  <customwidget>
   <class>InfoText</class>
   <extends>QLabel</extends>
   <header>widgets/InfoText.h</header>
  </customwidget>
  <customwidget>
   <class>SmartLabel</class>
   <extends>QLabel</extends>
   <header>widgets/SmartLabel.h</header>
   <slots>
    <signal>changedSystemOfMeasurementOrScale(PreviousScaleInfo)</signal>
    <slot>popContextMenu(QPoint)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>SmartLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/SmartLineEdit.h</header>
   <slots>
    <signal>textModified()</signal>
    <slot>lineChanged()</slot>
    <slot>lineChanged(PreviousScaleInfo)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
</ui>
